### DISCUSSION TITLE
WebP support, why/why not and how?

### DISCUSSION THREAD

**Discussion by srcnalt** [atteneder/glTFast#606] [Q&A]
Hi Andreas,

I know you have it in docs that WebP will not be supported however I was wondering what would be the underlying reason in detail. Compared to JPEG and PNG it's a well performing web native format, maybe not as good as KTX however still a good option.

I would be very happy if you could give me a lead on how I could bring support for WebP on a fork for test purposes.

**Comment by krisrok** [comment]
Hi @srcnalt, I've added WebP support [here](https://github.com/atteneder/glTFast/pull/632), feel free to check it out.

**Comment by srcnalt** [comment]
Thanks @krisrok will give it a try!
